src_c/event.c

```diff
@@ -173,7 +173,36 @@ pg_event_filter(void *_, SDL_Event *event)
     else if (type == PGE_KEYREPEAT) {
         event->type = SDL_KEYDOWN;
     }
-
+    else if (type == SDL_MOUSEWHEEL) {
+        // Generate a MouseButtonDown event for compatibility.
+        // https://wiki.libsdl.org/SDL_MouseWheelEvent
+        SDL_Event newevent;
+        newevent.type = SDL_MOUSEBUTTONDOWN;
+        
+        int x, y;
+        VIDEO_INIT_CHECK();
+        SDL_GetMouseState(&x, &y);
+        newevent.button.x = x;
+        newevent.button.y = y;
+        
+        newevent.button.state = SDL_PRESSED;
+        newevent.button.clicks = 1;
+        
+        /* 4/5 for roll up/down
+		 TODO: Wheel for x axis?
+		if (event->wheel.x > 0)
+            newevent.button.button = 4;
+        else if (event->wheel.x < 0)
+            newevent.button.button = 5;
+        */
+        if (event->wheel.y > 0)
+            newevent.button.button = 4;
```

Comment:
> You will no longer receive SDL_BUTTONDOWN events for mouse wheels, and buttons 4 and 5 are real mouse buttons now.

https://wiki.libsdl.org/MigrationGuide

How might one disambiguate these buttons from scrolling? A boolean "wheel" attribute in the event dict?